source/edge-server/permissions.txt

```diff
@@ -0,0 +1,274 @@
+Edge Server Permissions Guide
+=============================
+
+.. meta::
+  :keywords: code example
+   :description: Learn how to configure permissions for Edge Servers and client connections.
+
+.. facet::
+   :name: programming_language
+   :values: json
+
+.. facet::
+   :name: genre 
+   :values: reference
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Edge Server data access is governed by the intersection of:
+
+- Role-based permissions
+- User authentication
+- The query used to sync data to the Edge Server instance
+- If using Atlas Device SDK, the query used to sync data with the client
+
+Edge Server supports role-based permissions at the document and field level. 
+This page explores high-level Edge Server and client permissions configuration,
+and provides examples to illustrate basic permissions principles.
+
+Related documentation:
+
+- If you are new to role-based permissions in Atlas, get an overview of 
+  the permissions engine at :ref:`permissions`.
+- For information about how to define roles and permissions, refer to 
+  :ref:`define-roles-and-permissions`.
+- Edge Server uses Device Sync to sync data between the Edge Server instances
+  and Atlas. Edge Server and client roles must use 
+  :ref:`Device Sync-compatible permissions <device-sync-compatibility>`.
+
+.. _edge-server-permissions:
+
+Edge Server Permissions
+~~~~~~~~~~~~~~~~~~~~~~~
+
+An Edge Server instance is a user, similar to other incoming client 
+connections. The Edge Server must authenticate with Atlas, and it has its 
+own permissions to determine what data the instance can read and write.
+
+You can configure rules to generally apply to all Edge Servers in your service.
+Or you can get more granular with rules to specify a subset of data that
+the Edge Server instance can read and write.
+
+All Edge Servers
+````````````````
+
+You can create a role that applies to all Edge Server instances by specifying
+an ``applyWhen`` :ref:`rule expression <expressions>` where the 
+:ref:`%%user_type <rule-expression-user-expansion>` is ``edge``:
+
+.. code-block:: json
+
+   {
+     "name": "edgeServerRole",
+     "apply_when": {
+       "%%user.type": {
+         "$eq": "edge"
+       }
+     },
+     ...customize the permissions for all Edge Server instances...
+   }
+
+For example, if your data set does not contain sensitive data, such as a 
+weather service, you might add a role stating that Edge Servers can read and 
+write all data:
+
+.. code-block:: json
+
+   {
+     "name": "readAndWriteAll",
+     "apply_when": {
+       "%%user.type": {
+         "$eq": "edge"
+       }
+     },
+     "document_filters": {
+       "read": true,
+       "write": true
+     },
+     "insert": true,
+     "delete": true,
+     "search": true,
+     "read": true,
+     "write": true
+   }
+
+A Specific Edge Server
+``````````````````````
+
+If your data set contains sensitive information that should not sync to
```

Comment:
not just sensitive data, IMO. the most common use case is that you dont want the edge server to get all the data because its too much data for just one (storefront, ship, whatever).

Reply:
i guess since this is a permissioning doc, its fair to call it sensitive. feel free to ignore. but, per my comment below, users, could use permissioning to automatically make every edge server see a subset of the data - effectively avoiding having to set the syncing query on each edge server